spec/features/log_in_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.feature 'User logs in', type: :feature do
+  let(:user) { create(:user) }
+  before do
+    visit dashboard_path
+    click_link 'Sign In'
+  end
+
+  scenario 'with valid credentials' do
+    expect(page).to have_content 'Log In'
```

Comment:
**Critical - Test Quality**
Please refactor this to adhere to the [4 phases of testing](https://robots.thoughtbot.com/four-phase-test).

The expectation here can be done in its own scenario.